vardict.pl

```diff
@@ -2517,13 +2538,13 @@ sub realignlgins30 {
 	next unless( $p >= $START - $EXTENSION && $p <= $END + $EXTENSION );
 	push(@tmp5, [$p, $sc5v, $sc5v->{cnt}]);
     }
-    @tmp5 = sort {$b->[2] <=> $a->[2];} @tmp5;
+    @tmp5 = sort {$b->[2] <=> $a->[2] || $a->[0] <=> $b->[0];} @tmp5;
```

Comment:
Added additional sort by position (processing order may affect the following positions). The same in realignlgdel() and realignlgins().